specification/frontdoor/resource-manager/readme.md

```diff
@@ -28,17 +28,18 @@ These are the global settings for the FrontDoor API.
 title: FrontDoorManagementClient
 description: FrontDoor Client
 openapi-type: arm
-tag: package-2019-10
+tag: package-2019-11
```

Comment:
is `package-2019-10` released before?

Reply:
no, package-2019-10 and package-2019-11 were for some reason both created just several days ago.
it anything was released using one of these tags, package would be invalid.